src/shogun/mathematics/linalg/internal/implementation/MeanEigen3.h

```diff
@@ -73,11 +75,24 @@ struct int2float<int32_t>
 template <>
 struct int2float<int64_t>
 {
+	/** float64_t type **/
 	using floatType = float64_t;
 };
- 
+#else
```

Comment:
@karlnapf aha sorry I missed this..Because int2float type is used in Redux.h and MeanEigen3.h several times as return type. If I hide int2float from SWIG, it's going to be undefined.